utilities/input_tools.py

```diff
@@ -251,13 +282,72 @@ def getPOInames(rtfile, poi_type="mu"):
         impacts = rtfile['nuisance_impact_nois'].to_hist()
         names.append('Wmass')
 
-    if len(names)==0:
-        logger.warning('No free parameters found (neither signal strenght(s), nor W mass)')
-        return [None]
-
     return names
-    
-def readImpacts(rtfile, group, sort=True, add_total=True, stat=0.0, POI='Wmass', normalize=True):
+
+def readImpacts(fitresult_file, group, sort=True, add_total=True, stat=0.0, POI='Wmass', normalize=True):
+    if isinstance(fitresult_file, h5py.File):
+        impacts, labels, norm, total = readImpactsH5(fitresult_file, group, POI=POI)
+    else:
+        impacts, labels, norm, total = readImpactsRoot(fitresult_file, group, POI=POI)
+
+    if sort:
+        order = np.argsort(impacts)
+        impacts = impacts[order]
+        labels = labels[order]
+
+    if add_total:
+        impacts = np.append(impacts, total)
+        labels = np.append(labels, "Total")
+
+    if normalize:
+        impacts /= norm
```

Comment:
What's the norm?

Reply:
This is for the masked channel POIs where norm is the xsec, to be able to get the relative impact